src/SFA.DAS.EmployerAccounts/Services/ContentApiClient.cs

```diff
@@ -29,10 +29,15 @@ public async Task<string> Get(string type, string applicationId)
             await AddAuthenticationHeader();
 
             string banner = string.Empty;
+            var uri = string.Empty;
             switch (type)
```

Comment:
is this switch necessary? Always doing the same thing.

Reply:
I was thinking the same, was just following the pattern. Can take it out